src/Azure.Deployments.Extensibility.Core/Azure.Deployments.Extensibility.Core.csproj

```diff
@@ -20,7 +21,7 @@
 
   <ItemGroup>
     <PackageReference Update="Microsoft.SourceLink.GitHub" Version="8.0.0" />
-    <PackageReference Update="Nerdbank.GitVersioning" Version="3.6.133" />
+    <PackageReference Update="Nerdbank.GitVersioning" Version="3.6.139" />
```

Comment:
It seems the PackageReference in all the *.csproj have `Update` instead of `Include`. I think we have a package reference elsewhere in one of the shared files that needs to be updated.